File: 백준/Silver/18110. solved．ac/solved．ac.js
const input = require('fs').readFileSync(0).toString().trim().split('\n');

const n = Number(input[0]);

if (n === 0) {
    console.log(0);
    process.exit();
}

const scores = [];
for (let i = 1; i <= n; i++) {
    scores.push(Number(input[i]));
}
scores.sort((a, b) => a - b);

const cut = Math.round(n * 0.15);

let sum = 0;
for (let i = cut; i < n - cut; i++) {
    sum += scores[i];
}

const count = n - (2 * cut);
const average = Math.round(sum / count);

console.log(average);

File: 백준/Silver/18110. solved．ac/solved．ac.py
import sys

def my_round(num):
    if num - int(num) >= 0.5:
        return int(num) + 1
    else:
        return int(num)

input = sys.stdin.readline
n = int(input())

if n == 0:
    print(0)
else:
    opinions = []
    for _ in range(n):
        opinions.append(int(input()))
    
    opinions.sort()
    cut = my_round(n * 0.15)
    trimmed_list = opinions[cut : n-cut]
    print(my_round(sum(trimmed_list) / len(trimmed_list)))